Added devcontainer (#94)
DiamondLightSource/sci-react-ui: readme.md

Code can be found at [github.com/DiamondLightSource/sci-react-ui](https://github.com/DiamondLightSource/sci-react-ui).

> You'll need `pnpm` installed to build it. It is automatically installed as a feature when using the devcontainer.

Reviewer: ```suggestion
You'll need `pnpm` installed to build it (note: it is automatically installed as a feature if you are using devcontainers)
```
